website/components/remote-plugin-docs/utils/resolve-nav-data.js

```diff
@@ -139,8 +140,14 @@ async function resolvePluginEntryDocs(pluginConfigEntry, currentPath) {
     version,
     pluginTier,
     sourceBranch = 'main',
+    zipFile = '',
```

Comment:
It seems like we want to keep the `zipFile` option from being used in production - things like the "Edit" links could end up broken, for example.

If this is the case, I think we should add a quick validation check in `.github/workflows/check-plugin-docs.js`. Maybe something like:

```js
      // Validate pluginTier property
      /* ... etc ... */
      // Validate that local zip files are not used in production
      if (typeof pluginEntry.zipFile !== "undefined") {
        throw new Error(
          `Local ZIP file being used for "${
            title || pluginEntry.path || repo
          }". The zipFile option should only be used for local development. Please omit the zipFile attribute and ensure the plugin entry points to a remote repository.`
        );
      }
```

Reply:
awesome!